python/lsst/ap/verify/dataset.py

```diff
@@ -277,7 +277,10 @@ def makeCompatibleRepoGen3(self, repoDir):
         """
         # No way to tell makeRepo "create only what's missing"
         try:
-            repoConfig = dafButler.Butler.makeRepo(repoDir)
+            initConfig = dafButler.Config()
+            # Checksums greatly slow importing of large repositories
+            initConfig[".datastore.checksum"] = False
```

Comment:
Inside butler we prefer to write this as `initConfig["datastore", "checksum"]` rather than relying on the delimiter at all. You can of course keep it with the leading delimiter.

Reply:
If you prefer that format, consider emphasizing it in the class docs. Right now they focus almost entirely on delimiters, and all the alternatives are presented as if they were equal.